spyderlib/baseconfig.py

```diff
@@ -179,9 +179,9 @@ def add_image_path(path):
         return
     global IMG_PATH

```

Comment:
Nothing to do with this PR, but why is this line necessary? `IMG_PATH` will be taken from the outer namespace. It would be necessary if there was `IMG_PATH = something`, but here we're only `.append()`ing to it.